src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -57,6 +92,30 @@ public void bindDrivebaseControls() {
 		driveController.back().onTrue(new InstantCommand(s.drivebaseSubsystem::resetPose));
 	}
 
+	public void bindArmControls() {
+		armManualControl.toggleOnTrue(
+				new ManualArmOverrideCommand(
+						s.armSubsystem, codriveController::getRightY, codriveController::getLeftY));
+		armLowButton.onTrue(
```

Comment:
Wrist fully retracted may mean we lose game pieces after picking them up.  We should talk to the drive team to see the preferred wrist position for this button.  This doesn't need to be done before merging this.